[2024-09-17T23:28:38.669+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion scheduled__2024-09-17T22:00:00+00:00 [queued]>
[2024-09-17T23:28:38.689+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion scheduled__2024-09-17T22:00:00+00:00 [queued]>
[2024-09-17T23:28:38.689+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 0
[2024-09-17T23:28:38.717+0000] {taskinstance.py:1382} INFO - Executing <Task(SSHOperator): raw_ingestion> on 2024-09-17 22:00:00+00:00
[2024-09-17T23:28:38.723+0000] {standard_task_runner.py:57} INFO - Started process 1445 to run task
[2024-09-17T23:28:38.726+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion', 'raw_ingestion', 'scheduled__2024-09-17T22:00:00+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion.py', '--cfg-path', '/tmp/tmpy_59__fo']
[2024-09-17T23:28:38.730+0000] {standard_task_runner.py:85} INFO - Job 116: Subtask raw_ingestion
[2024-09-17T23:28:38.948+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion.raw_ingestion scheduled__2024-09-17T22:00:00+00:00 [running]> on host airflow-worker
[2024-09-17T23:28:39.291+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion' AIRFLOW_CTX_TASK_ID='raw_ingestion' AIRFLOW_CTX_EXECUTION_DATE='2024-09-17T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-17T22:00:00+00:00'
[2024-09-17T23:28:39.292+0000] {ssh.py:141} INFO - Creating ssh_client
[2024-09-17T23:28:39.293+0000] {ssh.py:114} INFO - ssh_hook is not provided or invalid. Trying ssh_conn_id to create SSHHook.
[2024-09-17T23:28:39.344+0000] {base.py:73} INFO - Using connection ID 'spark_ssh_default' for task execution.
[2024-09-17T23:28:39.345+0000] {ssh.py:300} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-09-17T23:28:39.378+0000] {transport.py:1893} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-09-17T23:28:39.570+0000] {transport.py:1893} INFO - Authentication (password) successful!
[2024-09-17T23:28:39.571+0000] {ssh.py:478} INFO - Running command: export SPARK_HOME=/usr/local/spark; /opt/conda/bin/spark-submit --master local[*] --executor-memory 2g /opt/shared/scripts/raw_ingestion.py
[2024-09-17T23:28:45.034+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO SparkContext: Running Spark version 3.3.1
[2024-09-17T23:28:45.069+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-09-17T23:28:45.329+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO ResourceUtils: ==============================================================
[2024-09-17T23:28:45.331+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-09-17T23:28:45.332+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO ResourceUtils: ==============================================================
[2024-09-17T23:28:45.332+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO SparkContext: Submitted application: load
[2024-09-17T23:28:45.362+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-09-17T23:28:45.376+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO ResourceProfile: Limiting resource is cpu
[2024-09-17T23:28:45.378+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-09-17T23:28:45.468+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO SecurityManager: Changing view acls to: root
[2024-09-17T23:28:45.469+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO SecurityManager: Changing modify acls to: root
[2024-09-17T23:28:45.469+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO SecurityManager: Changing view acls groups to: 
[2024-09-17T23:28:45.470+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO SecurityManager: Changing modify acls groups to: 
[2024-09-17T23:28:45.471+0000] {ssh.py:524} INFO - 24/09/17 23:28:45 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(root); groups with view permissions: Set(); users  with modify permissions: Set(root); groups with modify permissions: Set()
[2024-09-17T23:28:46.144+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 INFO Utils: Successfully started service 'sparkDriver' on port 43091.
[2024-09-17T23:28:46.226+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 INFO SparkEnv: Registering MapOutputTracker
[2024-09-17T23:28:46.296+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 INFO SparkEnv: Registering BlockManagerMaster
[2024-09-17T23:28:46.344+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-09-17T23:28:46.345+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-09-17T23:28:46.376+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-09-17T23:28:46.443+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-9d69e908-a3d0-4ca8-aefe-c17de5f2f020
[2024-09-17T23:28:46.536+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-09-17T23:28:46.582+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-09-17T23:28:46.996+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-09-17T23:28:46.999+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2024-09-17T23:28:47.001+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2024-09-17T23:28:47.001+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2024-09-17T23:28:47.002+0000] {ssh.py:524} INFO - 24/09/17 23:28:46 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2024-09-17T23:28:47.002+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2024-09-17T23:28:47.002+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 WARN Utils: Service 'SparkUI' could not bind on port 4046. Attempting port 4047.
[2024-09-17T23:28:47.020+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO Utils: Successfully started service 'SparkUI' on port 4047.
[2024-09-17T23:28:47.599+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO Executor: Starting executor ID driver on host spark-master
[2024-09-17T23:28:47.656+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-09-17T23:28:47.769+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 45059.
[2024-09-17T23:28:47.771+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO NettyBlockTransferService: Server created on spark-master:45059
[2024-09-17T23:28:47.774+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-09-17T23:28:47.783+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, spark-master, 45059, None)
[2024-09-17T23:28:47.789+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO BlockManagerMasterEndpoint: Registering block manager spark-master:45059 with 434.4 MiB RAM, BlockManagerId(driver, spark-master, 45059, None)
[2024-09-17T23:28:47.792+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, spark-master, 45059, None)
[2024-09-17T23:28:47.812+0000] {ssh.py:524} INFO - 24/09/17 23:28:47 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, spark-master, 45059, None)
[2024-09-17T23:28:50.392+0000] {ssh.py:524} INFO - 24/09/17 23:28:50 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-09-17T23:28:50.430+0000] {ssh.py:524} INFO - 24/09/17 23:28:50 INFO SharedState: Warehouse path is 'file:/root/spark-warehouse'.
[2024-09-17T23:28:56.177+0000] {ssh.py:524} INFO - 24/09/17 23:28:56 WARN MetricsConfig: Cannot locate configuration: tried hadoop-metrics2-s3a-file-system.properties,hadoop-metrics2.properties
[2024-09-17T23:28:56.249+0000] {ssh.py:524} INFO - 24/09/17 23:28:56 INFO MetricsSystemImpl: Scheduled Metric snapshot period at 10 second(s).
[2024-09-17T23:28:56.250+0000] {ssh.py:524} INFO - 24/09/17 23:28:56 INFO MetricsSystemImpl: s3a-file-system metrics system started
[2024-09-17T23:29:06.326+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 178, in execute
    result = self.run_ssh_client_command(ssh_client, self.command, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 163, in run_ssh_client_command
    exit_status, agg_stdout, agg_stderr = self.hook.exec_ssh_client_command(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/hooks/ssh.py", line 549, in exec_ssh_client_command
    raise AirflowException("SSH command timed out")
airflow.exceptions.AirflowException: SSH command timed out
[2024-09-17T23:29:06.333+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=raw_ingestion, task_id=raw_ingestion, execution_date=20240917T220000, start_date=20240917T232838, end_date=20240917T232906
[2024-09-17T23:29:06.375+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 116 for task raw_ingestion (SSH command timed out; 1445)
[2024-09-17T23:29:06.396+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-17T23:29:06.481+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
